jupyter_ai_personas/pr_review_persona/ci_tools.py

```diff
@@ -0,0 +1,126 @@
+import os
+import json
+from agno.tools import Toolkit
+from agno.utils.log import logger
+import re
+from github import Github
+import requests
+from agno.agent import Agent
+
+class CITools(Toolkit):
```

Comment:
nit: Not sure if we need a toolkit, as there is only one function and can be simplified into a custom tool.
https://docs.agno.com/tools/tool-decorator.

Also, should we name this `CILogsTool`?